lib/helpers.js

```diff
@@ -123,6 +125,17 @@ export const parseResults = (editor, results, filePath, showIgnored) => {
   return toReturn;
 };
 
+export const saveCleanedStyles = async (editor, results) => {
+  const result = results._postcssResult;
+  const styles = result.root.toString(result.opts.syntax);
+
+  if (styles === editor.getText()) {
+    // break to prevent infinite loop
```

Comment:
so just to make sure that I understand, it sounds like you're suggesting that I should grab the editor.getText() inside lint() and save it as say, `lastRunText` and then on line 118 where I set the fix flag with `fix: this.attemptFix && lastRunText !== editor.getText()` so that it doesn't attempt to fix if the files are unchanged?

Is there a preferred way to save state? I imagine it's poor form to just let a variable at the top of provideLinter() ?